src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -631,6 +633,13 @@ public ColumnStatisticsModel retrieveColumnStatistics(
     }
   }
 
+  public String manualExclusiveLock(AuthenticatedUserRequest userReq, UUID datasetId) {
+    return jobService
+        .newJob("Create manual exclusive lock on dataset.", DatasetLockFlight.class, null, userReq)
+        .addParameter(JobMapKeys.DATASET_ID.getKeyName(), datasetId)
+        .submit();
```

Comment:
That might be helpful, especially if the flight fails for whatever reason.  Also, looking ahead to using this from the UI, it might be nice to not need a job poller for this call